code/src/layouts/app-wrapper/tests/pageWrapper.test.tsx

```diff
@@ -0,0 +1,14 @@
+import PageWrapper from "@/layouts/app-wrapper/PageWrapper";
+import { render, screen } from "@testing-library/react";
+
+describe("PageWrapper", () => {
+  test("should render children", () => {
+    render(
+      <PageWrapper>
+        <div>Test children</div>
```

Comment:
Maybe we can get rid of `<div>` to improve performance

Reply:
Fixed!